code/src/Components/NinthQuestion.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export const NinthQuestion = ({ infinitiveVerb, setInfinitiveVerb }) => {
+  const handleNinthAnswer = (event) => {
+    setInfinitiveVerb(event.target.value);
+  }
+  return (
+    <>
+      <p className="instructionText"> Write an infititive verb</p>
```

Comment:
Great idea to use different classNames for the p's. Makes it so much easier to style. Will def keep that in mind for coming projects.